scanner/cmd/scannerctl/sbom.go

```diff
@@ -0,0 +1,85 @@
+package main
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"log"
+
+	"github.com/spf13/cobra"
+	v4 "github.com/stackrox/rox/generated/internalapi/scanner/v4"
+	"github.com/stackrox/rox/pkg/scannerv4/client"
+	"github.com/stackrox/rox/scanner/cmd/scannerctl/authn"
+	"github.com/stackrox/rox/scanner/indexer"
+)
+
+func sbomCmd(ctx context.Context) *cobra.Command {
```

Comment:
Nitpick: Worth mentioning the image will be indexed, if not already. Also, might be interesting to allow the creation of SBOMs without forcing an indexing operation (to ensure we are retrieving a already scanned/indexed image).

Reply:
> Also, might be interesting to allow the creation of SBOMs without forcing an indexing operation (to ensure we are retrieving a already scanned/indexed image).

If the image is already indexed then the existing index report will be used (current state). I could see value in adding a flag that will preven' re-indexing if an index report is not found? Thoughts?

Reply:
Updated desc of the cmd and added `skip-indexing` flag.